blockchain-agent-proof/pom.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>com.sphereon.alfresco.blockchain-agent</groupId>
+        <artifactId>blockchain-agent-parent</artifactId>
+        <version>0.0.2-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>blockchain-agent-proof</artifactId>
+
+    <packaging>jar</packaging>
+
+    <properties>
+        <blockchain-proof-sdk-java8.version>0.10.0</blockchain-proof-sdk-java8.version>
+        <easy-blockchain-lib.version>0.1.3</easy-blockchain-lib.version>
+        <crypto-keys-sdk-java8-okhttp-gson.version>0.9.0</crypto-keys-sdk-java8-okhttp-gson.version>
```

Comment:
Hmzz, not sure why we have this?

Reply:
Same. I'll prune the dependencies some more.

Reply:
Removed